setup.py

```diff
@@ -20,13 +20,13 @@
         "pyahocorasick>=1.1.6",
         "pycountry>=17.9.23",
         "unidecode>=1.0.22",
+        "parameterized>=0.6.1",
```

Comment:
Why are we switching `parameterized` to a regular dependency?

Reply:
Because our current infrastructure doesn't handle test-only dependencies well :(

Reply:
Interesting, so this is the case across the board?